services/movie.py

```
@@ -1,9 +1,11 @@
 from django.db.models import QuerySet
+from django.db import transaction
 
 from db.models import Movie
 
 
 def get_movies(
+    title: str = None,
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
```

Issue: Using mutable default arguments (`genres_ids: list[int] = None`, `actors_ids: list[int] = None`) is discouraged. While you use `None` as default, the type hint suggests a list, which may be misleading. Consider updating the type hints to `Optional[list[int]]` for clarity and to match the default value.